ocs_ci/ocs/ui/views.py

```
@@ -1126,6 +1126,70 @@
+    "create_storage_cluster_btn": (
+        "//button[normalize-space()='Create storage cluster']"
+        " | //button[contains(@data-test,'create-storage-cluster')]",
+        By.XPATH,
+    ),
+    # Per-row node checkboxes — id="node-<uuid>", class="pf-v6-c-check__input".
+    # There is no "Select all" header checkbox on this page; each row must be
+    # clicked individually.
+    "node_row_checkboxes": (
+        "//tbody//tr//input[@type='checkbox' and contains(@id,'node-')]"
+        " | //tbody//tr//input[@type='checkbox' and contains(@class,'pf-v6-c-check__input')]"
+        " | //tbody//tr//input[@type='checkbox']",
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)